### PR TITLE
typos

### PR DESCRIPTION
**Pull request by TApplencourt**

### CODE REVIEW

File: xprof/xprof.rb.in [hunk 1]
```diff
@@ -478,7 +478,7 @@ def setup_lttng(backends)
     channel_name = 'non-blocking-channel'
     exec("lttng enable-channel --userspace --session=#{lttng_session_uuid} #{channel_name}")
     exec("lttng add-context    --userspace --session=#{lttng_session_uuid} --channel=#{channel_name} -t vpid -t vtid")
-    exec("lttng enable-event  --userspace --session=#{lttng_session_uuid} --channel=#{channel_name} lttng_ust_sampling:*")
+    exec("lttng enable-event   --userspace --session=#{lttng_session_uuid} --channel=#{channel_name} lttng_ust_sampling:*")
```
Comment (bd4) [inline xprof/xprof.rb.in:481]: Is this change really needed, looks like just an extra space?

**Review by bd4 — CHANGES_REQUESTED** [review]

File: xprof/xprof.rb.in (hunk 1 above)
Comment (TApplencourt) [reply xprof/xprof.rb.in:481]: yeah, just to align with the other :)

**Review by bd4 — APPROVED** [review]

File: xprof/xprof.rb.in (hunk 1 above)
Comment (bd4) [reply xprof/xprof.rb.in:481]: Oh! makes sense.
